sklearn/multioutput.py

```diff
@@ -37,12 +39,90 @@ def _fit_estimator(estimator, X, y, sample_weight=None):
     return estimator
 
 
+def _partial_fit_estimator(estimator, X, y, classes=None, sample_weight=None,
+                           first_time=True):
+    if first_time:
+        estimator = clone(estimator)
+    else:
+        estimator = copy.copy(estimator)
```

Comment:
This is an interesting implementation detail. I acknowledge that copying like this is the only easy way to maintain consistent behaviour whether or not parallelism is used. However I think it should be a `deepcopy` if anything. And I suppose you should ensure this behaviour in tests: that across multiple calls to `partial_fit` the same estimator objects are not maintained.